wagtail/wagtailadmin/static_src/wagtailadmin/scss/core.scss

```diff
@@ -434,7 +434,7 @@ footer,
     }
 
     footer {
-        width: 80%;
+        width: calc(100% - #{$menu-width} - 100px); // [full width] - [width of nav menu] - [2 * width of margin]
```

Comment:
Yep, it needs to be created so we enforce the same value here, and for `margin-right` and `margin-left` 🙂.